### PR TITLE
Update blockchain npm packages

### PR DESCRIPTION
**Pull request by daniel-arnauer**
Update all npm packages to newest
Implement breaking changes

Closes parts of https://github.com/openkfw/TruBudget/issues/1089

### CODE REVIEW

File: blockchain/check-backup.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ stream.on("finish", async () => {
 });
 
 const loadConfig = (path) => {
-  const config = yaml.safeLoad(fs.readFileSync(path, "utf8"));
+  const config = yaml.load(fs.readFileSync(path, "utf8"));
```
Comment (mayrmartin) [inline blockchain/check-backup.js:105]: pls do not replace `load` by` safeLoad` more information [here](http://adilapapaya.com/docs/js-yaml/#safeloadstringoptions)

File: blockchain/src/index.js [hunk 2]
```diff
@@ -350,9 +355,7 @@ app.post("/chain", async (req, res) => {
           config.DirectoryHash,
           extractPath,
         );
-        const chainConfig = yaml.safeLoad(
-          fs.readFileSync(chainConfigPath, "utf8"),
-        );
+        const chainConfig = yaml.load(fs.readFileSync(chainConfigPath, "utf8"));
```
Comment (mayrmartin) [inline blockchain/src/index.js:358]: pls do not replace `load` by` safeLoad` more information [here](http://adilapapaya.com/docs/js-yaml/#safeloadstringoptions)

File: blockchain/src/index.js [hunk 3]
```diff
@@ -324,7 +329,7 @@ app.get("/readiness", (req, res) => {
 });
 
 const loadConfig = (path) => {
-  const config = yaml.safeLoad(fs.readFileSync(path, "utf8"));
+  const config = yaml.load(fs.readFileSync(path, "utf8"));
```
Comment (mayrmartin) [inline blockchain/src/index.js:332]: pls do not replace `load` by` safeLoad` more information [here](http://adilapapaya.com/docs/js-yaml/#safeloadstringoptions)

**Review by mayrmartin — CHANGES_REQUESTED** [review]

File: blockchain/check-backup.js (hunk 1 above)
Comment (daniel-arnauer) [reply blockchain/check-backup.js:105]: this docs are deprecated (and not even versioned lol). 
The current version of js-yaml is 4.x.x, so this applies: 
https://github.com/nodeca/js-yaml/blob/master/migrate_v3_to_v4.md

**Review by mayrmartin — APPROVED** [review]

**Pull request merged**